app/components/VotingCriteria/VotingParticipantInfo.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import { View, Text } from 'react-native';
+
+import { LocalizationContext } from '../Translations';
+import { FontFamily } from '../../assets/stylesheets/theme/font';
+
+import Participant from '../../models/Participant'
+import { getVotingParticipants } from '../../helpers/voting_criteria_helper';
+import uuidv4 from '../../utils/uuidv4';
+
+class VotingParticipantInfo extends Component {
+  static contextType = LocalizationContext;
+
+  _renderSummaryItem(value, index) {
```

Comment:
@limkimsan consider to pass group_name instead of index, in e.g: _renderSummaryItem(value, group_name)